### ISSUE TITLE
conda install on Centos 6.5 installs v1.0

### ISSUE THREAD

**Issue by davidmam** [dib-lab/sourmash#642]
Installing via conda 4.4.11 installs sourmash v 1.0
-bash-4.1$ conda list | grep sourmash
sourmash                  1.0                      py35_0    bioconda
-bash-4.1$
rather than the latest version.

**Comment by luizirber** [comment]
Did you set up the [bioconda channels](https://bioconda.github.io/#set-up-channels):
```
conda config --add channels defaults
conda config --add channels bioconda
conda config --add channels conda-forge
```
and then run `conda install sourmash`?

I've seen this happening before, but for users that tried to use the [anaconda instructions](https://anaconda.org/bioconda/sourmash): `conda install -c bioconda sourmash` doesn't work because it's missing the `conda-forge` channel...

**Comment by davidmam** [comment]
That did the trick - it was missing conda-forge. May I suggest these instructions be added to the README?

Thankyou

..d

Dr David Martin
Senior Lecturer in Bioinformatics
College of Life Sciences
University of Dundee


________________________________

**Comment by dsouzadaniel** [comment]
I'm working on this issue

**Comment by luizirber** [comment]
fixed by #677
